contrib/node/src/python/pants/contrib/node/tasks/node_task.py

```diff
@@ -17,13 +19,17 @@
 from pants.contrib.node.targets.node_test import NodeTest
 
 
+logger = logging.getLogger(__name__)
+
+
 class NodeTask(Task):
 
   @classmethod
   def subsystem_dependencies(cls):
     return super(NodeTask, cls).subsystem_dependencies() + (NodeDistribution.Factory,)
 
-  @memoized_property
+  @property
+  @memoized_method
```

Comment:
On my own machine, memorized_property will be marked as grammar error by pylint because it is not treated as a property. Does your pylint raise this error or did you abandon this rule?